index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <h2>Hello world, we are software developers!</h2>
+    <div id="containerpage">
+      <header id="header">
+        <div class="hlogo">
+          <img
+            id="header-img"
+            src="images/triallogo.png"
+            ;
+            alt="original trombones logo"
+          />
+        </div>
+
+        <nav id="nav-bar">
+          <ul>
+            <li><a class="nav-link" href="#Features">Features</a></li>
+            <li><a class="nav-link" href="#How It Works">How It Works</a></li>
+            <li><a class="nav-link" href="">Pricing</a></li>
+          </ul>
+        </nav>
+      </header>
+      <div style="margin-top: 100px; margin-left: 50px">
+        <iframe
+          width="560"
+          height="315"
+          src="https://www.youtube.com/embed/y8Yv4pnO7qc"
+          title="Roman Carnival Overture Op. 9 for Five Trombones"
+          frameborder="0"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+          allowfullscreen
+        ></iframe>
+      </div>
+      <form id="form" action="https://www.freecodecamp.com/email-submit">
+        <input
```

Comment:
Validations are support essential for client-side users and server-side owners to interchange valid data please add all expected  validation for all input element
```suggestion
        <input
```

Reply:
This one is better, you're asking to add validations (clear) and stating that they are essential for the client and server. Good job! 🤓 🚀